### PR TITLE
Chore: add autogen CLI reference doc script

### PR DESCRIPTION
**Pull request by wonderflow**
Signed-off-by: Jianbo Sun <jianbo.sjb@alibaba-inc.com>

### Description of your changes

refer to https://github.com/oam-dev/kubevela.io/pull/418

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: hack/docgen/gen.go [hunk 1]
```diff
@@ -17,16 +17,64 @@ limitations under the License.
...
+	"path/filepath"
+	"strings"
 
 	"github.com/oam-dev/kubevela/references/cli"
 
 	"github.com/spf13/cobra/doc"
 )
 
 func main() {
+	rootPath := "../kubevela.io/docs/cli/"
```
Comment (barnettZQG) [inline hack/docgen/gen.go:32]: read root path from args and default is "../kubevela.io/docs/cli/"

**Review by leejanee — APPROVED** [review]
LGTM

**Review by hongchaodeng — APPROVED** [review]

File: hack/docgen/REDAME.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+2. Run generate command in kubevela root dir.
+
+```shell
+cd kubevela/
+go run ./hack/docgen/gen.go
+```
+
+3. Then you can check difference in kubevela.io.
```
Comment (hongchaodeng) [inline hack/docgen/REDAME.md:19]:
```suggestion
3. Then you can check the difference in kubevela.io.
```

File: hack/docgen/gen.go (hunk 1 above)
Comment (wonderflow) [reply hack/docgen/gen.go:35]: sure

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**